core/src/i18n/en.yaml

```
@@ -939,6 +944,7 @@ issuers:
     messages:
       failure: Failed to create the issuer.
       success: Issuer created.
+  description: <0>Issuer</0> is a Namespace resource responsible for provision of certificates.
```

```suggestion
  description: <0>Issuer</0> is a Namespace resource responsible for provisioning of certificates.
```

not sure, maybe "provisioning" ?